autoload/wintabs/ui.vim

```diff
@@ -176,13 +207,20 @@ function! s:get_spaceline()
   let line = g:wintabs_ui_sep_spaceline
   let length = 1
   for tab in range(1, spaces)
+    " get tab name
+    let name = s:get_tab_name(tab)
+
     " get and normalize space name
-    let name = ' '.tab.' '
-    let length += len(name)
+    " 2 is added to length of name because we will add two separators around
+    " name
+    let length += (len(name) + 2)
```

Comment:
It's not guaranteed that user would use single character for separators, so let's put this logic `let length += len(name)` after each condition below (after line 220 and line 223).